src/main/java/com/spotify/docker/client/messages/Ulimit.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2015 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.docker.client.messages;
+
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+
+import java.util.Objects;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+
+@JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
+public class Ulimit {
+  @JsonProperty("Name") private String name;
+  @JsonProperty("Soft") private Integer soft;
+  @JsonProperty("Hard") private Integer hard;
+
+  public Ulimit() {
+  }
+
+  private Ulimit(final Builder builder) {
+    this.name = builder.name;
+    this.soft = builder.soft;
+    this.hard = builder.hard;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public String name() {
+    return name;
+  }
+
+  public Integer soft() {
+    return soft;
+  }
+
+  public Integer hard() {
+    return hard;
+  }
+
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+
+    final Ulimit that = (Ulimit) o;
+
+    return Objects.equals(this.name, that.name) &&
+        Objects.equals(this.soft, that.soft) &&
+        Objects.equals(this.hard, that.hard);
+  }
+
+  @Override
+  public int hashCode() {
+        return Objects.hash(name, soft, hard);
+    }
+
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
+        .add("name", name)
+        .add("soft", soft)
+        .add("hard", hard)
+        .toString();
+  }
+
+  public static class Builder {
```

Comment:
I think so. `Ulimits` of a container are returned in the `HostConfig` on [container inspect](https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.18.md#inspect-a-container), but they can also be set to a desired value when [creating a container](https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.18.md#create-a-container).